### DISCUSSION TITLE
I Don't Know Create Customer Window like this window

### DISCUSSION THREAD

**Discussion by hubing53** [AvaloniaUI/Avalonia#8943] [Q&A] [answered]
![屏幕截图 2022-09-12 210746](https://user-images.githubusercontent.com/112156001/189662538-6eaeb459-f91a-45f9-a205-d9dc1b4fb6c8.png)

**Comment by timunie** [comment, marked as answer]
Probably this tutorial helps you: [Music Store Demo](https://docs.avaloniaui.net/tutorials/music-store-app)

Also please have a look at FluentAvalonia's CoreWindow. Maybe this is for you: [FluentAvalonia-Repo](https://github.com/amwx/FluentAvalonia)
Sample of CoreWindow: [Find it in the demo App](https://github.com/amwx/FluentAvalonia/blob/master/FluentAvaloniaSamples/Pages/FAControlPages/CoreWindowPage.axaml)

**Reply by hubing53** [reply]
You are right

**Comment by hubing53** [comment]
Thankyou In the Music Store Demo I Find the answer.
